pesign-obs-integration.spec

```diff
@@ -17,6 +17,10 @@
 # needssslcertforbuild
 
 
+%if (0%{?fedora})
+%global debug_package %{nil}
```

Comment:
does it work if you disable `<debuginfo><disable/></debuginfo> in the build project? I don't think it should be enabled for fedora builds

Reply:
It does not unfortunately, it's due to a Fedora-specific rpm specpart. I think it would if it was a noarch package (given it's just scripts), but I guess the ship sailed on that one by now

Reply:
okay, I see. fine.